README.rst

```diff
@@ -64,42 +73,43 @@ Additional optional arguments
- delimiter=<delimiter>         Specifies the delimiter to recognize in the input .csv file (e.g., "--delimiter=,"). If not specified, the script tries to automatically determine the delimiter. The special keyword "tab" can be used to indicate a tab-delimited csv.
- resume                        Starts the prediction from the point at which it was halted. If the prediction stopped, for example due to error or network connection issue, you can run the same command with all the same arguments plus this ``resume`` argument. If you do not include this argument, and the script detects a previous script was interrupted mid-execution, DataRobot prompts whether to resume. When resuming a script, you cannot change the ``dataset_filepath``,  ``model_id``, ``project_id``, ``n_samples``, or ``keep_cols``.
- help                          Show usage help for the command.
- fast                          *Experimental*: Uses a faster csv processor. Note that this method does not support multiline csv.
- stdout                        Send all log messages to stdout.
- auto_sample                   Override the ``<n_samples>`` value and instead use chunks of roughly 1.5 MB to improve throughput. On by default.
- encoding                      Declare the dataset encoding. If an encoding is not provided, the batch_scoring script attempts to detect it (e.g., "utf-8", "latin-1", or "iso2022_jp"). `See the Python docs for a list of valid encodings <https://docs.python.org/3/library/codecs.html#standard-encodings>`_.
- skip_dialect                  Tell the batch_scoring script to skip csv dialect detection.
-============================== ===========
+============================== ========== ========= ===========
+  Argument                     Standalone Dedicated Description
+------------------------------ ---------- --------- -----------
+ host=<host>                      \+         \+     Specifies the hostname of the prediction API endpoint (the location of the data from where to get predictions).
+ user=<user>                      \-         \+     Specifies the username used to acquire the api-token. Use quotes if the name contains spaces.
+ <import_id>                      \+         \-     Specifies imported model unique identification string. You can ask for ID your prediction administrator who did the import procedure.
+ <project_id>                     \-         \+     Specifies the project identification string. You can find the ID: embedded in the URL that displays when you are in the Leaderboard (for example, ``https://<host>/projects/<project_id>/models``) or when the prediction API is enabled, from the example shown when you click **Deploy Model** for a specific model in the Leaderboard.
```

Comment:
In the docs this is a bulleted list. If not a list change to the following. If it IS a list, you can leave as it.

You can find the ID embedded in the URL that displays when you are in the Leaderboard (for example, ``https://<host>/projects/<project_id>/models``). Alternatively, when the prediction API is enabled, the ID displays in the example shown when you click **Deploy Model** for a specific model in the Leaderboard.